pkgs/tools/package-management/appimagekit/default.nix

```diff
@@ -73,12 +79,9 @@ in stdenv.mkDerivation rec {
     glib zlib cairo openssl fuse
     xz inotify-tools libarchive
     squashfsTools makeWrapper
```

Comment:
makeWrapper belongs into nativeBuildInputs.

Reply:
good catch. should be fixed now.